pkg/Microsoft.Windows.Compatibility.Shims/Microsoft.Windows.Compatibility.Shims.pkgproj

```diff
@@ -5,41 +5,38 @@
   <PropertyGroup>
     <PackageVersion>$(CompatibilityShimsPackageVersion)</PackageVersion>
     <_ShimsLocationPath>$(BaseIntermediateOutputPath)shims/netcoreapp/facades/</_ShimsLocationPath>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <SkipValidatePackage>true</SkipValidatePackage>
   </PropertyGroup>
 
   <ItemGroup>
-    <ShimsToInclude Include="mscorlib.dll" />
-    <ShimsToInclude Include="System.dll" />
-    <ShimsToInclude Include="System.Configuration.dll" />
-    <ShimsToInclude Include="System.Core.dll" />
-    <ShimsToInclude Include="System.Data.dll" />
-    <ShimsToInclude Include="System.Drawing.dll" />
-    <ShimsToInclude Include="System.Net.dll" />
-    <ShimsToInclude Include="System.Security.dll" />
-    <ShimsToInclude Include="System.ServiceModel.Web.dll" />
-    <ShimsToInclude Include="System.ServiceProcess.dll" />
-    <ShimsToInclude Include="System.Transactions.dll" />
-    <ShimsToInclude Include="WindowsBase.dll" />
-  </ItemGroup>
 
-  <Target Name="AddShimsToPackage"
-          BeforeTargets="GetPackageFiles">
+    <File Include="$(PlaceHolderFile)">
+      <TargetPath>/lib/netstandard2.0</TargetPath>
+    </File>
+
+    <File Include="$(PlaceHolderFile)">
+      <TargetPath>/ref/netstandard2.0</TargetPath>
+    </File>
 
-    <Error Text="Required facade: %(ShimsToInclude.Identity) was not found in $(_ShimsLocationPath)." Condition="!Exists('@(ShimsToInclude->'$(_ShimsLocationPath)%(Identity)')')" />
+    <File Include="$(PlaceHolderFile)">
+      <TargetPath>/lib/netcoreapp2.1</TargetPath>
+    </File>
 
+    <File Include="$(PlaceHolderFile)">
+      <TargetPath>/ref/netcoreapp2.1</TargetPath>
+    </File>
+
+    <ProjectReference Include="..\..\src\shims\shims.proj" />
+    <ProjectReference Include="..\..\src\shims\manual\mscorlib.csproj" />
+    <ProjectReference Include="..\..\src\shims\manual\System.csproj" />
+    <ProjectReference Include="..\..\src\shims\manual\System.Data.csproj" />
+  </ItemGroup>
+
+  <Target Name="ClearDependencies" BeforeTargets="GenerateNuSpec">
```

Comment:
Did you try setting OmitDependencies=true?

Reply:
No, will try it out.